src/explorer/TimelineCredChart.js

```diff
@@ -48,14 +50,22 @@ export class TimelineCredChart extends React.Component<Props> {
   render() {
     const {timelineCred, displayedNodes} = this.props;
     const intervals = timelineCred.intervals();
+    let timelineCredViewInterval: Interval = {
+      startTimeMs: intervals[0].startTimeMs,
+      endTimeMs: intervals[intervals.length - 1].endTimeMs,
+    };
+    const timelineCredView = new TimelineCredView(
```

Comment:
Assuming we refactor the constructor to make the interval optional, then we shouldn't need to do the manual interval creation here.

Reply:
(You marked this resolved, but I think it still applies.)

Reply:
good catch